docs/reference/landmark-based-facial-expression-recognition.md

```diff
@@ -0,0 +1,450 @@
+following public methods:
+
+
+#### `ProgressiveSpatioTemporalBLNLearner` constructor
+```python
+ProgressiveSpatioTemporalBLNLearner(self, lr, batch_size, optimizer_name, lr_schedule,
+                                    checkpoint_after_iter, checkpoint_load_iter, temp_path,
+                                    device, num_workers, epochs, experiment_name,
+                                    device_ind, val_batch_size, drop_after_epoch,
+                                    start_epoch, dataset_name, 
+                                    blocksize, numblocks, numlayers, topology,
+                                    layer_threshold, block_threshold)
+```
+
+Constructor parameters:
+- **lr**: *float, default=0.1*  
+  Specifies the initial learning rate to be used during training.
+- **batch_size**: *int, default=128*  
+  Specifies number of samples to be bundled up in a batch during training. This heavily affects memory usage, adjust according to your system.
+- **optimizer_name**: *str {'sgd', 'adam'}, default='sgd'*  
+  Specifies the optimizer type that should be used.
+- **lr_schedule**: *str, default=' '*  
+  Specifies the learning rate scheduler.
+- **checkpoint_after_iter**: *int, default=0*
+  Specifies per how many training iterations a checkpoint should be saved. If it is set to 0 no checkpoints will be saved.
+- **checkpoint_load_iter**: *int, default=0*  
+  Specifies which checkpoint should be loaded. If it is set to 0, no checkpoints will be loaded.
+- **temp_path**: *str, default=''*  
+  Specifies a path where the algorithm saves the checkpoints and onnx optimized model (if needed).
+- **device**: *{'cpu', 'cuda'}, default='cuda'*  
+  Specifies the device to be used.
+- **num_workers**: *int, default=32*  
+  Specifies the number of workers to be used by the data loader.
+- **epochs**: *int, default=400*  
+  Specifies the number of epochs the training should run for.
+- **experiment_name**: *str, default='pstbln_casia'*  
+  String name to attach to checkpoints.
+- **device_ind**: *list, default=[0]*  
+  List of GPU indices to be used if the device is 'cuda'. 
+- **val_batch_size**: *int, default=256*  
+  Specifies number of samples to be bundled up in a batch during evaluation. This heavily affects memory usage, adjust according to your system.
+- **drop_after_epoch**: *list, default=[1000]*  
+  List of epoch numbers in which the optimizer drops the learning rate.  
+- **start_epoch**: *int, default=0*  
+  Specifies the starting epoch number for training.  
+- **dataset_name**: *str {'AFEW', 'CASIA', 'CK+'}, default='CASIA'*  
+  Specifies the name of dataset that is used for training and evaluation. 
+- **block_size**: *int, default=5*  
+  Specifies the number of output channels (or neurons) that are added to each layer of the network at each progression iteration. 
+- **numblocks**: *int, default=100*  
```

Comment:
This argument name format is not consistent with the others.
It should rather be `num_blocks` (same for `num_layers`).